tests/dvslib/dvs_common.py

```diff
@@ -16,7 +16,7 @@ class PollingConfig:
         strict: If the strict flag is set, reaching the timeout will cause tests to fail.
     """
 
-    polling_interval: float = 0.01
+    polling_interval: float = 0.1
```

Comment:
Try fix test case failed
Code change in this PR will not pops all entries within one Consumer::exec() API call, which may cause data base update delay.